src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java

```diff
@@ -648,6 +648,8 @@ public boolean equals(Object obj)
                 && Objects.equals(other.channelTypes, channelTypes)
                 && Objects.equals(other.minValue, minValue)
                 && Objects.equals(other.maxValue, maxValue)
+                && Objects.equals(other.required, required)
+                && Objects.equals(other.autoComplete, autoComplete)
```

Comment:
Since these are primitives, we don't need to use Objects#equals

```suggestion
                && other.required == required
                && other.autoComplete == autoComplete
```